lib/mercenary/presenter.rb

```diff
@@ -83,11 +85,11 @@ def command_presentation
     #
     # Returns the value of whatever function is called
     def method_missing(meth, *args, &block)
-      if meth.to_s =~ /^print_(.+)$/
-        send("#{$1.downcase}_presentation")
+      if meth.to_s =~ %r!^print_(.+)$!
+        send("#{Regexp.last_match(1).downcase}_presentation")
       else
         super # You *must* call super if you don't handle the method,
-              # otherwise you'll mess up Ruby's method lookup.
+        # otherwise you'll mess up Ruby's method lookup.
```

Comment:
think the following works better with rdoc..
```ruby
else
  # You *must* call super if you don't handle the method,
  # otherwise you'll mess up Ruby's method lookup.
  super
end
```